### ISSUE TITLE
[Issue]: Jellyfin doesn't respect preferred subtitle language, even when set to always play

### ISSUE THREAD

**Issue by szerencl** [jellyfin/jellyfin#9021]
### Please describe your bug

On a totally brand new install I have subtitles set to always play and selected English as the preferred language. I want English audio and English subtitles.
![kép](https://user-images.githubusercontent.com/57007996/211035672-3a50ec60-39d9-4667-8eab-0ef95d8a3f22.png)

However Jellyfin still doesn't respect the setting, it just selects Hungarian subtitles, or sometimes (I think when there is no Hungarian) the first available (Danish, Chinese etc.).
![kép](https://user-images.githubusercontent.com/57007996/211035786-9d0eccc4-922d-490f-b90b-852b623d12d6.png)![kép](https://user-images.githubusercontent.com/57007996/211035797-16a53d9b-8dde-4612-97c4-a182afb1dabc.png)

### Jellyfin Version

10.8.0

### if other:

10.8.8

### Environment

```markdown
- OS: Synology DiskStation Manager
- Virtualization: Docker
- Clients: Windows, LG WebOS, Android, Browser
- Browser: Firefox 108.0.1, Chrome 108
- FFmpeg Version: 5.1.2
- Playback Method: Direct Play
- Hardware Acceleration:
- Plugins:
- Reverse Proxy: none
- Base URL: none
- Networking: Host
- Storage: local
```

### Jellyfin logs

```shell
[2023-01-06 14:59:51.755 +00:00] [INF] [19] Jellyfin.Api.Helpers.MediaInfoHelper: StreamBuilder.BuildVideoItem( Profile="Jellyfin Media Player", Path="/media/Series/Loki/Loki.S01.1080p.DSNP.WEB-DL.DDPA5.1.H.264-TOMMY/Loki.S01E01.Glorious.Purpose.REPACK.1080p.DSNP.WEB-DL.DDP5.1.H.264-TOMMY.mkv", AudioStreamIndex=null, SubtitleStreamIndex=null ) => ( PlayMethod=DirectPlay, TranscodeReason=0 ) "media:/videos/5f9459d5-c2a0-c8c7-6e30-9ca47e2fa955/stream.mkv?MediaSourceId=5f9459d5c2a0c8c76e309ca47e2fa955&Static=true&VideoCodec=h264&AudioCodec=eac3&AudioStreamIndex=1&SubtitleStreamIndex=2&api_key=<token>&SubtitleMethod=Embed&Tag=245e41a324c52fe01e92e6d709504cd4"
[2023-01-06 14:59:52.494 +00:00] [WRN] [19] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from "<Server IP>" to "<Playback IP>" in 0:00:00.7283616 with Status Code 204
[2023-01-06 14:59:57.503 +00:00] [INF] [19] Emby.Server.Implementations.Session.SessionManager: Playback stopped reported by app "Jellyfin Media Player" "1.7.1" playing "Glorious Purpose". Stopped at "3750" ms
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by veoxer** [comment]
I have the exact same problem.

**Comment by karserasl** [comment]
Same problem.. 
Any update on this?

**Comment by 1337joe** [comment]
This should already be fixed by #8472 for the next release (10.9).
